### DISCUSSION TITLE
💡 자료공유 👋 건의사항 디스커션 카테고리 신설

### DISCUSSION THREAD

**Discussion by DaleSeo** [DaleStudy/leetcode-study#324] [공지사항]
스터디 멤버 분들께서 자료를 공유하거나 의견을 건의할 창구가 마땅하지 않다고 느끼시는지 자꾸 디스코드 서버의 "잡담" 채널로 관련 내용이 올라오는 것 같습니다. 사실 저희 디스코드 서버에서 리트코드 스터디에 참여하시는 분은 극히 일부라서, 스터디 관련 대화는 "리트코드-스터디" 채널에서 이루어져야 합니다. 그런데 디스코드 특성 상 과거 데이터 검색이 좀 어려운 편이라서 두고 두고 참고해야 하는 대화를 나누기에는 좀 부적합니다. 그래서 이러한 좋은 자료나 건의 내용이 장기적으로 좀 더 체계적으로 관리될 수 있도록 깃허브 디스커션에 관련 카테고리를 신설하였으니 활용 부탁드리겠습니다.

![Shot 2024-08-14 at 17 39 31@2x](https://github.com/user-attachments/assets/ea2420a5-4beb-4bc5-8722-904bad1862c1)

**Comment by obzva** [comment]
좋은 것 같습니다 ㅎㅎ
잘 몰라서 잡담 채널에 여쭸는데, 빠르게 카테고리 신설해주셔서 감사합니다
